packages/zoe/src/contractSupport/durability.js

```diff
@@ -36,40 +39,46 @@ harden(makeEphemeraProvider);
 /**
  *
  */
-export const makeEphemeralStoredSubscriberProvider = () => {
-  /** @type {WeakMap<Subscriber<any>, StoredSubscriber<any>>} */
+export const makeTopicMetaProvider = () => {
+  /** @type {WeakMap<Subscriber<any>, import('@agoric/notifier').TopicMeta<any>>} */
```

Comment:
Would this make more sense in `notifier.js`? Or is there something specific to contracts going on here?